src/melee/ft/chara/ftPopo/ftPp_SpecialS.c

```diff
@@ -790,8 +794,26 @@ void ftPp_SpecialS_80121164(Fighter_GObj* gobj)
     }
 }
 
-/// #ftPp_SpecialHi_Enter
+void ftPp_SpecialHi_Enter(Fighter_GObj* gobj)
+{
+    Fighter* fp = GET_FIGHTER(gobj);
+    ftIceClimberAttributes* da = fp->dat_attrs;
+
+    fp->gr_vel /= *(float*)((u8*)da + 0x84);
```

Comment:
Let's add a float-typed x84 member to ftIceClimberAttributes so we can just do `fp->gr_vel /= da->x84;` here